src/FluentHub.Uwp/UserControls/Overview/PullRequestOverview.xaml.cs

```diff
@@ -0,0 +1,104 @@
+﻿using FluentHub.Uwp.Services;
+using FluentHub.Uwp.ViewModels.UserControls.Overview;
+using Microsoft.Extensions.DependencyInjection;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Animation;
+using muxc = Microsoft.UI.Xaml.Controls;
+
+namespace FluentHub.Uwp.UserControls
+{
+    public sealed partial class PullRequestOverview : UserControl
+    {
+        #region propdp
+        public static readonly DependencyProperty ViewModelProperty =
+            DependencyProperty.Register(
+                nameof(ViewModel),
+                typeof(PullRequestOverviewViewModel),
+                typeof(PullRequestOverviewViewModel),
+                new PropertyMetadata(null));
+
+        public PullRequestOverviewViewModel ViewModel
+        {
+            get => (PullRequestOverviewViewModel)GetValue(ViewModelProperty);
+            set
+            {
+                SetValue(ViewModelProperty, value);
+                if (ViewModel is not null)
+                    SelectItemByTag(ViewModel.SelectedTag);
+            }
+        }
+        #endregion
+
+        public PullRequestOverview()
+        {
+            InitializeComponent();
+        }
+
+        private void OnPullRequestNavViewItemInvoked(muxc.NavigationView sender, muxc.NavigationViewItemInvokedEventArgs args)
+        {
+            var service = App.Current.Services.GetRequiredService<INavigationService>();
+
+            switch (args.InvokedItemContainer.Tag.ToString().ToLower())
+            {
+                default:
+                case "conversation":
+                    service.Navigate(
+                        typeof(Views.Repositories.Codes.CodePage),
+                        new Models.FrameNavigationArgs()
+                        {
+                            Login = ViewModel.PullRequest.Repository.Owner.Login,
```

Comment:
I guess, you can make this initialization before switch. You'll save a lines of code and it would be more readable.

Reply:
Yes, I will update the notifications wrapper.